src/main/java/com/dutact/web/common/message/ErrorMessage.java

```diff
@@ -0,0 +1,14 @@
+package com.dutact.web.common.message;
```

Comment:
Suggestion: move to `com.dutact.web.common.web` or `com.dutact.web.common.http` or `com.dutact.web.common.api`